### DISCUSSION TITLE
erorr nextcloud office

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#1669] [Questions] [answered]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. open a dokument with nc office
2. see the error

### Expected behavior 
open the dokument
### Actual behavior 
error

### Host OS 
Ubuntu Server 22.04

#### Nextcloud AIO version 
latest stable
#### Current channel 
stable
#### Other valuable info 
![grafik](https://user-images.githubusercontent.com/90777102/210566406-36809f0c-1fdc-4002-b77e-503fad10a9a5.png)

**Comment by szaimen** [comment]
Hi, can you follow https://github.com/nextcloud/all-in-one/discussions/1358?

**Comment by ghost** [comment, marked as answer]
work with ,0.0.0.0/0

**Comment by szaimen** [comment]
okay, moving to discussion then.

**Comment by ghost** [comment]
Since this is not a permanent solution, how can I solve the problem?

**Reply by szaimen** [reply]
you need to find out which ip-address you need to put into that field. I cannot help you with that unfortunately as I don't know your network.

**Comment by ghost** [comment]
Strangely enough, the problem fixed itself after the daily backup.

**Reply by szaimen** [reply]
Interesting!
